manifests/ubuntu/agent5.pp

```diff
@@ -67,8 +69,9 @@
     location => $location,
     release  => $release,
     repos    => $repos,
-    require  => Package['apt-transport-https'],
-    notify   => [Exec['datadog_apt-get_remove_agent6'],Exec['apt_update']],
+    key      => $key,
+    require  => Exec['apt-transport-https'],
```

Comment:
As mentioned for A6, I would like to keep `ensure_packages()`, unless it's 100% clear that we cannot, and just remove this `require`.